Initial release

mrjutsu opened the pull request.

mrjutsu committed:
Initial commit

mrjutsu committed:
Require Laravel 8.0 as the minimum version

mrjutsu committed:
GItignore and license

mrjutsu committed:
Composer file updated to use correct dependencies

mrjutsu committed:
Install command

mrjutsu committed:
Autoload changed in composer file

mrjutsu committed:
Service provider and command registration

mrjutsu committed:
Syntax error in composer file

mrjutsu committed:
Change autoload to use psr-4. Use correct service provider namespace

mrjutsu committed:
Update Install command

mrjutsu committed:
Update Install command

Additional line break when adding the Telescope key to the .env file.

Changed the called vendor:publish command.

mrjutsu committed:
Change report.txt path

mrjutsu committed:
Use correct path for report.txt

mrjutsu committed:
Fixes report.txt insertion

mrjutsu committed:
Improvements to Install command

Use only one line break before the Telescope key.

Publish missing Telescope migrations

mrjutsu committed:
Updates dependencies versions

mrjutsu committed:
Improves Install command

Added info text when publishing Telescope's migrations.

Use magic variable to load report.txt file

mrjutsu committed:
Adds additional comment to Install command

mrjutsu committed:
Organizes migrations in the Install command

mrjutsu committed:
Adds readme

mrjutsu committed:
Updates readme

mrjutsu committed:
Adds views to the Install command

mrjutsu committed:
Fixes path for report.txt

mrjutsu committed:
Fixes paths

mrjutsu committed:
Runs migrations organizer after publishing Fortify assets

mrjutsu committed:
Register views with Fortify

mrjutsu committed:
Fixes indentation in the txt files. Adds missing closing brackets.

mrjutsu committed:
Updates Install command

Refactors code to use constants for the directories check and creations instead of strings

mrjutsu committed:
Published Sass assets

Fixes paths

mrjutsu committed:
Refactores Install command

Uses Illuminate's Filesystem

mdahlke commented:
@mrjutsu @wisnetmark @bkolstad 

I think we publish what we have now so we can test requiring in a new site. 
We can tag this as `0.0.1-dev`

Thoughts?

wisnetmark commented:
@mdahlke @mrjutsu @bkolstad 

i'm on board with that

mrjutsu committed:
Add packages to package.json

If the package.json file doesn't exist, it is created

mrjutsu committed:
Updated Install command and package.json. Added webpack.mix.js

Renamed devDependencies variable. Added additional node dependencies.

mrjutsu committed:
Publish front-end assets

Refactors in the Install command.
Creates js directory if it doesn't exist.
Adds assets necessary for Vue and overall JS.

mrjutsu committed:
Updated installation success message

mrjutsu committed:
Corrected syntax error

mrjutsu committed:
Corrected variable when copying front-end assets

mrjutsu committed:
Corrected command's name

mrjutsu committed:
Changed $name to $signature

mrjutsu committed:
Overwrite front-end assets

mrjutsu committed:
Added Laravel Dusk

mrjutsu committed:
Added warn message after installation of the starter kit

mrjutsu committed:
Readme updated

mrjutsu committed:
Corrected dependencies added to package.json

mrjutsu committed:
Updated list of requirements in Readme

mrjutsu committed:
Removed unnecessary dependency. Modified webpack.mix.js

mrjutsu committed:
Removed unused line

mrjutsu committed:
Added npm required version in Readme
